src/app/services/constraint-mapping.service.ts

```diff
@@ -122,7 +122,6 @@ export class ConstraintMappingService {
     if (constraint.concept.encryptionDescriptor && constraint.concept.encryptionDescriptor.encrypted) {
       // todo: children IDs implementation
       item.encrypted = true;
-      item.queryTerm = this.cryptoService.encryptIntegerWithCothorityKey(constraint.concept.encryptionDescriptor.id);
```

Comment:
I think the encrypted field is not used anymore. Maybe if you want to clean this to, this is the right PR?

Reply:
Indeed, I'll look into removing it